packages/core/test/test-util/test-services/v4/multiple-schemas-service/TestEnumType2.ts

```diff
@@ -4,7 +4,17 @@
  * This is a generated file powered by the SAP Cloud SDK for JavaScript.
  */
 
+/**
+ * This enum represents the enum type "[[A_TestEnumType2]]".
+ * The members represent values of EDM type Edm.Int32.
```

Comment:
I am not really getting why the underlying data type is important? As far as I can see in the request the user will use the string key and this goes also to the the request like this. So `Gender eq 'Female'` not  `Gender eq 0`.